sys/include/net/nanocoap.h

```diff
@@ -224,6 +224,9 @@ typedef struct {
     const char *path;               /**< URI path of resource               */
     coap_method_flags_t methods;    /**< OR'ed methods this resource allows */
     coap_handler_t handler;         /**< ptr to resource handler            */
+#ifdef MODULE_GCOAP
+    uint8_t flags;                  /**< flags e.g. for separate response   */
+#endif
     void *context;                  /**< ptr to user defined context data   */
 } coap_resource_t;
```

Comment:
This can likely be merged wlth methods, also to avoid API breakage. There is precedent for that (MATCH_SUBTREE).

Reply:
Good idea, I changed it like that. I didn't do it at first because I thought it's not a coap method